everything-true/everything-true.js

```diff
@@ -0,0 +1,11 @@
+function truthCheck(collection, pre) {
+  // Is everyone being true?
+  for (let obj of collection) {
```

Comment:
can you try using this https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every